Commonalities/documentation/Glossary.md

```diff
@@ -1,11 +1,12 @@
+
 # Glossary
 
 | Term      | Description | Usage in API (Parameter/Field) | Alternative developer-friendly terms |
 | ------------ | ----------- | ----------- |  ----------- |
-| **Device** | End-user equipment able to connect to a mobile network. Examples of devices include smartphones or IoT sensors/actuators.   | device |  |  |
+| **Device** | End-user equipment able to connect to a mobile network. Examples of devices include smartphones or IoT sensors/actuators.   | device |  |  |
 | **Application Server** | A server hosting backend applications to deliver some business logic to clients. | applicationServer |  |  |
 | **Phone Number** |A public identifier addressing a telephone subscription. In mobile networks it corresponds to the MSISDN (Mobile Station International Subscriber Directory Number). In order to be globally unique it has to be formatted in international format, according to E.164 standard.  | phoneNumber |  |  |
-| **Network Access Identifier** |  A public identifier addressing a subscription in a mobile network. In 3GPP terminology, it corresponds to the GPSI formatted with the External Identifier ({Local Identifier}@{Domain Identifier}). Unlike the telephone number, the network access identifier is not subjected to portability ruling in force, and is individually managed by each operator.| networkAccessIdentifier  |  |  |
+| **Network Access Identifier** |  A public identifier addressing a subscription in a mobile network. In 3GPP terminology, it corresponds to the GPSI formatted with the External Identifier ({Local Identifier}@{Domain Identifier}). Unlike the telephone number, the network access identifier is not subjected to portability ruling in force, and is individually managed by each operator.| networkAccessIdentifier  |  |  |
```

Comment:
Network Access Identifier (NAI) is actually an IETF standard (latest: https://datatracker.ietf.org/doc/rfc7542/, previous https://datatracker.ietf.org/doc/rfc4282) and also used in fixed networks (e.g. with RADIUS). 3GPP TS 23.003 is referring to the RFC 4282. Therefore I propose:

```suggestion
| **Network Access Identifier** |  A public identifier addressing a subscription in a network, defined in [IETF RFC 7542](https://datatracker.ietf.org/doc/rfc7542/). In 3GPP mobile networks it is one option for the General Public Subscriber Identifier (GPSI) formatted with the External Identifier ({Local Identifier}@{Domain Identifier}). Unlike the phone number, the network access identifier is not subjected to portability ruling in force, and is individually managed by each operator.| networkAccessIdentifier  |  |  |
```